### PR TITLE
[ZH] Update `Ranking_Criteria\osu!`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!/zh.md [hunk 1]
```diff
@@ -1,207 +1,201 @@
...
-## 总体
-
-总体规定和指南适用于各种 osu! 难度。节奏相关的指南适用于 180 BPM 左右的 4/4 拍谱面。如果你的歌曲的 BPM 比 180 快很多或者慢很多，一些数值会不一样，[缩放 BPM](/wiki/Ranking_Criteria/Scaling_BPM)中有更详细的介绍。
+#### 规定
 
-### 通用
+- **物件在 4:3 游玩比例下不能超出屏幕。** 即使物件只有一部分超出屏幕，也会造成读图困难。请测试你的谱面来确认合规。
+- **转盘必须足够长以在开启 Auto 游玩时能转出至少 1000 奖励分。** 更短的转盘并没有足够的供玩家旋转的时长。
+- **除非强制使用默认皮肤，谱面必须设置至少两种不同的自定义 combo 颜色。** 任何情况下，combo 颜色组不能与谱面的背景图片、故事板、背景视频相混淆。这保证了玩家能看清所有物件，并避免使用了自定义 combo 颜色组的物件不会凑巧与背景元素相混淆。
+- **请勿在 hitnormal 音效较弱时使用[键盘音](/wiki/Beatmapping/Hitsound#keysound)。** 否则玩家不能获得足够的反馈。
```
Comment (PercyDan54) [inline wiki/Ranking_Criteria/osu!/zh.md:18]:
```suggestion
- **请勿在 hitnormal 音效较弱时使用[打击音效](/wiki/Beatmapping/Hitsound#keysound)。** 否则玩家不能获得足够的反馈。
```

File: wiki/Ranking_Criteria/osu!/zh.md [hunk 2]
```diff
@@ -1,207 +1,201 @@
...
+- **请确保 hitnormal 音效不会与歌曲内的音色过于相似。** 否则玩家不能获得足够的反馈。
+- **请确保 osu! 难度的转谱不会破坏其他模式的星数或表现分系统。** 在某些情况下，部分 `.osu` 文件转换为其他模式的谱面时会错误地格式化，导致转谱显示出与实际难度不匹配的星数，游玩后也会获得与实际难度不匹配的表现分。
```
Comment (PercyDan54) [inline wiki/Ranking_Criteria/osu!/zh.md:20]: 其他好像很多地方都是保留不翻

```suggestion
- **请确保 osu! 难度的转谱不会破坏其他模式的星数或pp系统。** 在某些情况下，部分 `.osu` 文件转换为其他模式的谱面时会错误地格式化，导致转谱显示出与实际难度不匹配的星数，游玩后也会获得与实际难度不匹配的 pp。
```

Comment (YumeMuzi) [reply wiki/Ranking_Criteria/osu!/zh.md:20]: 原文就是表现分，，，

Comment (cdwcgt) [reply wiki/Ranking_Criteria/osu!/zh.md:20]: 因为中文的 [Performance points](https://github.com/ppy/osu-wiki/blob/master/wiki/Performance_points/zh.md) 里面是用的 `pp` 而不是 `表现分`
所以还是改了吧
说实话这样很多文章都要统一

Comment (Walavouchey) [reply wiki/Ranking_Criteria/osu!/zh.md:20]: lowercase is fine unless it's necessary due to the language

**Comment by Walavouchey** [comment]
please only tick the relevant checklist item *after* you've applied reviews or when someone has approved the translation on github. the same goes for your other pull requests. i've unticked them for you again.

**Review by cdwcgt — CHANGES_REQUESTED** [review]
RC大写

File: wiki/Ranking_Criteria/osu!/zh.md (hunk 2 above)
Comment (YumeMuzi) [reply wiki/Ranking_Criteria/osu!/zh.md:20]: ok lowercase `pp` applied

**Pull request merged**